GNUmakefile

```diff
@@ -20,7 +20,7 @@ doc:
 
 #### Unit tests and sanity check #############################################
 
-LAMBDAPI     = 'dune exec -- lambdapi'
+LAMBDAPI     = dune exec -- lambdapi
```

Comment:
Is this variable declaration still useful?

Reply:
I dunno, I guess you can keep it until unless you want to port the tests to dune.